klaus/contrib/auto_klaus.py

```diff
@@ -0,0 +1,103 @@
+"""
+Alternative take on the "automatically discovered repositories" concept
+that requires no threads, polling or inotify. Instead the filesystem is
+consulted whenever a repository name is looked up.
+
+Since os.path.exists() and os.listdir() are fairly quick filesystem
+operations, performance should be good for small to medium sites.
+FancyRepo() objects are cached.
+
+Repositories are identified by the existence of a
+
+    <reponame>/git-daemon-export-ok
+
+file (for compatibility with gitweb).
+
+Example usage:
+
+    from klaus.contrib.auto_klaus import Klaus, SlashDynamicRepos
+
+    application = Klaus('/srv/git', "My git repositories", False)
+
+    application.wsgi_app = httpauth.AlwaysFailingAuthMiddleware(
```

Comment:
Most people using klaus won't be able to understand how to actually run a klaus server with this code. We should have something as simple as this: https://github.com/jonashaag/klaus/wiki/Autoreloader

Reply:
Sure, I'll look into that.

Reply:
Added a `wsgi_autodetect.py` in the style of `wsgi_autoreload.py`.